.github/workflows/benchmarks.yml

```
@@ -118,11 +141,21 @@ jobs:
 
       - name: Run benchmarks autotuned
         if: ${{ inputs.benchmarkAgents == '' }}
-        run: make run-benchmark-autotuned index-benchmark-results
+        run: make run-benchmark-autotuned
 
       - name: Run benchmarks self tuned
         if: ${{ inputs.benchmarkAgents != '' }}
-        run: make run-benchmark index-benchmark-results
+        run: make run-benchmark
+
+      - name: Cat standalone server logs
+        if: ${{ failure() }}
+        run: |
+          if [ "$RUN_STANDALONE" = true ]; then
+            make cat-apm-server-logs
+          fi
```

step native condition

```suggestion
        if: ${{ failure() && env.RUN_STANDALONE == 'true' }}
        run: make cat-apm-server-logs
```